flarefly/fitter.py

```diff
@@ -240,6 +240,20 @@ def __build_background_pdfs(self, obs):
 
                 else:
                     Logger(f'Missing datasample for Kernel Density Estimation of background {ipdf}!', 'FATAL')
+            elif 'pol' in pdf_name:
```

Comment:
move it before the `kde` implementations and add the option `chebpolN` in the docstring